### DISCUSSION TITLE
Broken subresources within forwarded Codespace webserver

### DISCUSSION THREAD

**Discussion by jyasskin** [community/community#97305] [Codespaces]
### Select Topic Area

Bug

### Body

I have a webserver at https://vigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev/ for which the main page redirects through github.dev/pf-signin and loads, but then subresources _also_ try to redirect through github.dev/pf-signin, and the (module) script subresources fail with:

> Access to script at 'https://github.dev/pf-signin?id=majestic-lake-q9jmm2x&cluster=usw2&name=vigilant-space-fortnight-97jq4r6p7fp9v9&port=4321&pb=https%3A%2F%2Fvigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev%2Fauth%2Fpostback%2Ftunnel%3Frd%3D%252Fsrc%252Flayouts%252FLayout.astro%253Fastro%2526type%253Dstyle%2526index%253D0%2526lang.css%26tunnel%3D1' (redirected from 'https://vigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev/src/layouts/Layout.astro?astro&type=style&index=0&lang.css') from origin 'https://vigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

Did something change in the last day or two? This is with Edge 120.0.2210.133 in case it varies by browser.

Added:

On some reloads, the main page fails with:

> Refused to send form data to 'https://vigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev/auth/postback/tunnel?tunnel=1' because it violates the following Content Security Policy directive: "form-action 'self' github.com gist.github.com objects-origin.githubusercontent.com vigilant-space-fortnight-97jq4r6p7fp9v9-4321.app.github.dev".

**Comment by djlin95** [comment]
I'm getting a similar error - can't launch the web app through the forwarded port with the same error on Chrome.  Just happened.

**Comment by djlin95** [comment]
Looks like its back for me.

**Reply by jyasskin** [reply]
The set of subresources that get redirected seems to vary somewhat randomly on each reload. When I run the non-dev version of the server (so it has browser caching enabled), it eventually loads everything, but of course that doesn't work for the dev version.

**Reply by djlin95** [reply]
Actually, spoke to soon - seemed like it worked only once.  It's not working again.

**Comment by jyasskin** [comment]
It seems to be fixed for now. The subresources don't redirect through github.dev anymore.

**Reply by djlin95** [reply]
Yes, I just noticed that right now too  - I have been waiting this whole time crossing my fingers.  I wonder what happened.   Thanks Jeffrey.
